qemu/tests/openflow_acl_test.py

```diff
@@ -95,20 +96,13 @@ def access_service(access_sys, access_targets, disabled, host_ip,
                         out = out_err
                 else:
                     try:
-                        out = run_func(access_cmd, timeout=op_timeout)
-                        stat = 0
+                        stat, out = run_func(access_cmd, timeout=op_timeout)
                         check_string = access_params.get("check_from_output")
                         if check_string and check_string in out:
                             stat = 1
-                    except aexpect.ShellCmdError as err:
-                        out = err.output
-                        stat = err.status
                     except aexpect.ShellTimeoutError as err:
                         out = err.output
                         stat = 1
-                        session.close()
-                        session = vm.wait_for_login(timeout=timeout)
-                        run_func = session.cmd
```

Comment:
Why remove this part? Cause if hit timeout exception, to close the session and reopen it seems make sense.

Reply:
Hi @vivianQizhu , I remove this part because it's a local variable, and there are some shadows name in outer scope.
So may encounter this error `local variable 'session' referenced before assignment`(1769624#c1)
Although we can name these `session` separately, the code is a bit confusing, so I don't want to change it for now.

Reply:
@PaulYuuu Get your point, anyway the case will fail in this condition, so both okay for me.